test/unit/SwaggerCombine.spec.js

```diff
@@ -591,7 +602,8 @@ describe('[Unit] SwaggerCombine.js', () => {
         ]);
       });
 
-      it('throws an error if path name already exists', () => {
+      it('throws an error if path name already exists and opts propery continueOnConflictingPaths is false', () => {
+        instance.opts = {continueOnConflictingPaths: false};
```

Comment:
You don't need to set `continueOnConflictingPaths: false` explicitly, as it is falsy (i.e. `undefined`) by default.